testkit/examples/server/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+/npm-debug.log
```

Comment:
Personally I prefer to have a single `.gitignore` at the repository root. What do you think?

Reply:
I don't have a particularly strong opinion on the topic. IMO, "local" `.gitignore`s can be more understandable and easier to maintain.

Reply:
I don't have strong arguments, but I have the exactly opposing opinion. :upside_down_face: As for me, it is harder to maintain several files.

@aleksuss @ozkriff @vitvakatu @anyone
Opinions?

Reply:
Same thing, one gitignore to rule the world